Add note about ActiveSupport to CONTRIBUTING.md [ci-skip] (#46440)
rails/rails: CONTRIBUTING.md

* Do not open an issue on GitHub until you have collected positive feedback about the change. GitHub issues are primarily intended for bug reports and fixes.

> * We generally reject changes to Active Support, especially when they introduce more changes to core classes in Ruby like Array and Hash. This change should be proposed to Ruby instead, as we don't want to conflict with future versions of Ruby.  
>     The feature/issues tracker for Ruby is [here](https://bugs.ruby-lang.org/issues) and the contributing guides are [here](https://docs.ruby-lang.org/en/3.1/contributing_rdoc.html)
> 

Reviewer: ```suggestion
* We generally reject changes to Active Support core extensions. Those change should be proposed in the [Ruby issue tracker instead](https://bugs.ruby-lang.org/issues), as we don't want to conflict with future versions of Ruby.  
```

Author: Fixed!
